### DISCUSSION TITLE
Retiring old discussions

### DISCUSSION THREAD

**Discussion by pp-mo** [SciTools/iris#4536] [Ideas]
It seems it isn't possible to 'close' a Discussion -- only _delete_ it.
Which does not retain the old contributions for reference, e.g. as we would do on an Issue.

## Propose 
re-categorise as "Archived"

**Comment by pp-mo** [comment]
Created new "Archived" category
https://github.com/SciTools/iris/discussions/categories/archived

I've already done this with some existing ones :
  * https://github.com/SciTools/iris/discussions/4186
  * https://github.com/SciTools/iris/discussions/4071
  * https://github.com/SciTools/iris/discussions/4072

I have also put a note in the head of each one, like this ....
> # ARCHIVED
> _was_ category = Releases  🚩
> 
> # Original...

Which also records what category it _used_ to belong to.

**Comment by bjlittle** [comment]
Sounds all a bit meta... but can we archive this archive thread?

Seems like a done deal @pp-mo :wink:

**Comment by pp-mo** [comment]
Don't need this any more I think : we can now **_close_** issues.

**Reply by bjlittle** [reply]
Yup, you totally can. No need to archive now 👍
